lib/AST/Module.cpp

```diff
@@ -426,8 +427,8 @@ void Module::lookupMember(SmallVectorImpl<ValueDecl*> &results,
       return VD->getModuleContext() == this;
     });
 
-    const DeclContext *accessScope = nominal->getFormalAccessScope();
-    if (accessScope && !accessScope->isModuleContext())
+    auto DC = nominal->getFormalAccessScope().getDeclContext();
+    if (DC && !DC->isModuleContext())
```

Comment:
I think this code is really saying `accessScope.isPrivate() || accessScope.isFileScope()`. That's a little more expressive than checking the associated DC.